README.md

```diff
@@ -129,15 +129,16 @@ If the runner is not able to access github.com, any Nodejs versions requested du
 ## Advanced usage
 
 1. [Check latest version](docs/advanced-usage.md#check-latest-version)
-2. [Using a node version file](docs/advanced-usage.md#node-version-file)
-3. [Using different architectures](docs/advanced-usage.md#architecture)
-4. [Using nigthly versions](docs/advanced-usage.md#nightly-versions)
-5. [Using rc versions](docs/advanced-usage.md#rc-versions)
-6. [Caching packages data](docs/advanced-usage.md#caching-packages-data)
-7. [Using multiple operating systems and architectures](docs/advanced-usage.md#multiple-operating-systems-and-architectures)
-8. [Publishing to npmjs and GPR with npm](docs/advanced-usage.md#publish-to-npmjs-and-gpr-with-npm)
-9. [Publishing to npmjs and GPR with yarn](docs/advanced-usage.md#publish-to-npmjs-and-gpr-with-yarn)
-10. [Using private packages](docs/advanced-usage.md#use-private-packages)
+1. [Using a node version file](docs/advanced-usage.md#node-version-file)
```

Comment:
Could you please return numeration.

Reply:
Having markdown lists numerated is ant-pattern because it cause unnecessary diffs on every inserting/deleting item. MD renders are expected to increment numbers:
https://github.com/akv-platform/setup-node/tree/v-sdolin/v8-canary#advanced-usage

Reply:
I think we should keep previous numeration. Moreover, if we'd like to get rid of numeration it would be better to use points (-).

Reply:
fixed